app/javascript/mastodon/features/audio/index.jsx

```diff
@@ -22,6 +22,8 @@ const messages = defineMessages({
 const TICK_SIZE = 10;
 const PADDING   = 180;
 
+var currentAudio = null;
```

Comment:
Maybe it would make sense to have a `currentMedia` shared between audio/video components?

Reply:
Good call! Changing this soon 🤝